modules/admin-quota-overview.adoc

```diff
@@ -0,0 +1,327 @@
+|The total number of `ConfigMap` objects that can exist in the project.
+
+|`persistentvolumeclaims`
+|The total number of persistent volume claims that can exist in the project.
+
+|`openshift.io/imagestreams`
+|The total number of image streams that can exist in the project.
+
+|===
+
+You can configure an object count quota for these standard namespaced resource types using the `count/<resource>.<group>` syntax.
+
+[source,terminal]
+----
+$ oc create quota <name> --hard=count/<resource>.<group>=<quota> <1>
+----
+
+<1> `<resource>` is the name of the resource, and `<group>` is the API group, if applicable.
+Use the `kubectl api-resources` command for a list of resources and their associated API groups.
+
+== Setting resource quota for extended resources
+
+Overcommitment of resources is not allowed for extended resources, so you must specify `requests` and `limits` for the same extended resource in a quota.
+Currently, only quota items with the prefix `requests.` are allowed for extended resources. The following is an example scenario of how to set resource quota for the GPU resource `nvidia.com/gpu`.
+
+.Procedure
+
+. Determine how many GPUs are available on a node in your cluster. For example:
++
+[source,terminal]
+----
+$ oc describe node ip-172-31-27-209.us-west-2.compute.internal | egrep 'Capacity|Allocatable|gpu'
+
+.Example output
+
+                    openshift.com/gpu-accelerator=true
+Capacity:
+ nvidia.com/gpu:  2
+Allocatable:
+ nvidia.com/gpu:  2
+  nvidia.com/gpu  0           0
+----
++
+In this example, 2 GPUs are available.
+
+. Set a quota in the namespace `nvidia`. In this example, the quota is `1`:
++
+[source,terminal]
+----
+$ cat gpu-quota.yaml
+
+.Example output
+
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: gpu-quota
+  namespace: nvidia
+spec:
+  hard:
+    requests.nvidia.com/gpu: 1
+----
+
+. Create the quota:
++
+[source,terminal]
+----
+$ oc create -f gpu-quota.yaml
+
+.Example output
+
+resourcequota/gpu-quota created
+----
+
+. Verify that the namespace has the correct quota set:
++
+[source,terminal]
+----
+$ oc describe quota gpu-quota -n nvidia
+
+.Example output
+
+Name:                    gpu-quota
+Namespace:               nvidia
+Resource                 Used  Hard
+--------                 ----  ----
+requests.nvidia.com/gpu  0     1
+----
+
+. Run a pod that asks for a single GPU:
++
+[source,terminal]
+----
+$ oc create pod gpu-pod.yaml
+----
++
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: gpu-pod-
+  namespace: nvidia
+spec:
+  restartPolicy: OnFailure
+  containers:
+  - name: rhel7-gpu-pod
+    image: rhel7
+    env:
+      - name: NVIDIA_VISIBLE_DEVICES
+        value: all
+      - name: NVIDIA_DRIVER_CAPABILITIES
+        value: "compute,utility"
+      - name: NVIDIA_REQUIRE_CUDA
+        value: "cuda>=5.0"
+
+    command: ["sleep"]
+    args: ["infinity"]
+
+    resources:
+      limits:
+        nvidia.com/gpu: 1
+----
+
+. Verify that the pod is running:
```

Comment:
This suggestion applies to all steps containing command which misses the below piece.
```suggestion
. Verify that the pod is running by running the following command:
```

Reply:
Done, thank you!